docs/source/generics.rst

```diff
@@ -223,6 +223,26 @@ example we use the same type variable in two generic functions:
    def last(seq: Sequence[T]) -> T:
        return seq[-1]
 
+Note that a variable cannot have a type variable in its type unless the type variable is bound in a containing generic class or function:
```

Comment:
Please break the long line. Also, maybe drop the leading "Note that" -- the previous section also starts like that and it's a bit repetitive.